### ISSUE TITLE
Provide few options to show the nested tags in the main view

### ISSUE THREAD

**Issue by woodwm** [MuiseDestiny/zotero-style#197]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

Version `2.3.3` introduces a new feature to show only part of the nested tags by using regular expression. I think this may be too complex for most users. It would be more user friendly to provide few options to show the nested tags in the main view, such as: show only the deepest level for the nested tags, show the two deepest levels, show all levels.

**Describe alternatives you've considered**

**Additional context**

**Comment by Elodie-0117** [comment]
同意 我设置标签的方式也是例如#Audit/firm #Audit/parter #Audit/risk 这样的 所以展示的时候如果只有第二个单词 会有点奇怪 希望可以整个标签都展示出来 更清晰明了一些
虽然可以全部标签都使用 #Audit/Audit firm 这样的 却会导致标签太冗长

**Comment by MuiseDestiny** [comment]
> 同意 我设置标签的方式也是例如#Audit/firm #Audit/parter #Audit/risk 这样的 所以展示的时候如果只有第二个单词 会有点奇怪 希望可以整个标签都展示出来 更清晰明了一些 虽然可以全部标签都使用 #Audit/Audit firm 这样的 却会导致标签太冗长

本来就是可以的啊，match写一个#就可以了

**Comment by Elodie-0117** [comment]
> > 同意 我设置标签的方式也是例如#Audit/firm #Audit/parter #Audit/risk 这样的 所以展示的时候如果只有第二个单词 会有点奇怪 希望可以整个标签都展示出来 更清晰明了一些 虽然可以全部标签都使用 #Audit/Audit firm 这样的 却会导致标签太冗长
> 
> 本来就是可以的啊，match写一个#就可以了

哦 那我说的是左下角的标签窗口那里呢

**Comment by luohuaaa** [comment]
> > > 同意 我设置标签的方式也是例如#Audit/firm #Audit/parter #Audit/risk 这样的 所以展示的时候如果只有第二个单词 会有点奇怪 希望可以整个标签都展示出来 更清晰明了一些 虽然可以全部标签都使用 #Audit/Audit firm 这样的 却会导致标签太冗长
> > 
> > 
> > 本来就是可以的啊，match写一个#就可以了
> 
> 哦 那我说的是左下角的标签窗口那里呢

那你直接用原生的标签显示方式不就行了
